src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Utilities/EnvironmentHelper.cs

```diff
@@ -150,27 +151,32 @@ static ApplicationConfig ReadApplicationConfig (string envFile)
 						ret.jni_add_native_method_registration_attribute_present = ConvertFieldToBool ("jni_add_native_method_registration_attribute_present", envFile, i, field [1]);
 						break;
 
-					case 7: // bound_stream_io_exception_type: byte / .byte
+					case 7:
```

Comment:
Two questions:

1. Elsewhere, there's a comment on the `case` line saying what field it's for.  It's missing here.  Should there be a comment?
2. Why insert as "field #7" instead of appending as "field #12"?

Reply:
Why insert as "field #7"? So that all the `bool` fields are together in e.g. https://github.com/xamarin/xamarin-android/pull/6009/files#diff-920537864b4d4fbcacfad5ebfdcfd73470855176b83f4c4592d604a7057b7d3dR108